build.ps1

```diff
@@ -13,16 +13,12 @@ param(
     [Alias('s')]
     [string]$ToolsSource = 'https://aspnetcore.blob.core.windows.net/buildtools',
     [Parameter(ValueFromRemainingArguments = $true)]
-    [string[]]$MSBuildArgs
+    [string[]]$Arguments
 )
 
 $ErrorActionPreference = 'Stop'
 
-try {
-    Import-Module -Force -Scope Local $PSScriptRoot/files/KoreBuild/KoreBuild.psd1
-    Install-Tools $ToolsSource $DotNetHome
-    Invoke-RepositoryBuild $Path @MSBuildArgs
-}
-finally {
-    Remove-Module 'KoreBuild' -ErrorAction Ignore
-}
+$korebuildConsoleproj = "tools/KoreBuild.Console/KoreBuild.Console.csproj"
+
+& dotnet run -p $korebuildConsoleproj install-tools --toolsSource $ToolsSource --dotNetHome $DotNetHome $Arguments
```

Comment:
Regrettably, this won't work. It requires a pre-existing installation of dotnet, which we are not guaranteed to have. Something needs to execute via PowerShell/bash to install dotnet before you can run KoreBuild.Console.